src/AbstractFixer.php

```diff
@@ -34,7 +43,7 @@ public function __construct()
     {
         if ($this instanceof ConfigurableFixerInterface) {
             try {
-                $this->configure(null);
+                $this->configure(array());
```

Comment:
issue with `header_comment` fixed.
could you investigate memory issue when remembering the definition ?